src/gui/sdlmain.cpp

```diff
@@ -1763,10 +1764,15 @@ static void GUI_ShutDown(Section *)
 
 static void SetPriority(PRIORITY_LEVELS level)
 {
-	// TODO replace platform-specific API with SDL_SetThreadPriority
+	// Just let the OS scheduler manage priority
+	if (level == PRIORITY_LEVEL_DEFAULT)
+		return;
+
+		// TODO replace platform-specific API with SDL_SetThreadPriority
 #if defined(HAVE_SETPRIORITY)
-// Do nothing if priorties are not the same and not root, else the highest
-// priority can not be set as users can only lower priority (not restore it)
+	// Do nothing if priorties are not the same and not root, else the
+	// highest priority can not be set as users can only lower priority (not
+	// restore it)
```

Comment:
This comment.. 
 - Do **nothing** if
 - the priority is **not** the same and
 - **not** the root user

I wonder if we can replace the triple-negative: _"Priority can always be lowered but requires elevated rights to increase it"_

Reply:
```
	// Do nothing if priorties are not the same and not root, since priority 
	// can always be lowered but requires elevated rights to increase
```

?

Reply:
```
	// If the priorities are different, do nothing unless the user is root,
	// since priority can always be lowered but requires elevated rights 
	// to increase
```